CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
```

Comment:
I don't remember what error I saw before that caused me to link openssl. I unlinked it and the build still works fine on my machine. It would be useful to have a clean OS X machine for testing such as a Travis [multi-os build](https://docs.travis-ci.com/user/multi-os). I'll remove the openssl stuff from these instructions and we'll see if anyone else runs into it.